code/modules/antagonists/wizard/equipment/artefact.dm

```diff
@@ -110,6 +110,19 @@
 	eat()
 	return
 
+/obj/singularity/wizard/attack_tk(mob/user)
+	if(iscarbon(user))
+		var/mob/living/carbon/C = user
+		to_chat(C, "<span class='userdanger'>OH GOD! NONE OF ITS REAL! NONE OF IT IS REEEEEEEEEEEEEEEEEEEEEEEEAL!</span>")
+		//for(var/datum/component/mood/insaneinthemembrane in C.datum_components)
+		//	insaneinthemembrane.sanity = 0

```

Comment:
This kept runtiming, I couldn't figure out why. I also tried locate() before this.

Reply:
GET_COMPONENT_FROM(insaneinthemembrance, /datum/component/mood, C)

Reply:
although Ninja will tell you this is bad probably

Reply:
🙏